cms/templates/js/course-outline.underscore

```diff
@@ -235,6 +235,25 @@ if (is_proctored_exam) {
                         <% } %>
                     </p>
                 </div>
+                <% if (course.get('self_paced') && course.get('is_custom_relative_dates_active') && xblockInfo.get('relative_weeks_due')) { %>
+                    <div class="status-grading">
```

Comment:
Do you need to have this? Is it used for any selector?
It looks like `status-grading` is for the "standard" case whereas this one already has the div above with `status-timed-proctored-exam`. You can also see further down how there is a case with div `status-hide-after-due`

Reply:
yeah without that line the opacity effect (where the text lightens if you move away from it) part of `status-grading` and `status-timed-proctored-exam` doesn't seem to apply (while other subsection information like the grading type does have that effect...) Is that reasoning not a good enough reasoning to use this class :O

Reply:
So we do want to keep that. Hmm. What if you move this into the existing div and p tags and just have 
```
<% if (xblockInfo.get('due_date') && !course.get('self_paced')) { %>
...
<% else if (course.get('self_paced') && course.get('is_custom_relative_dates_active') && xblockInfo.get('relative_weeks_due')) { %>
<span class="icon fa fa-calendar" aria-hidden="true"></span>
...
```

Reply:
That puts everything on the same line:( 
![image](https://user-images.githubusercontent.com/60379333/127562986-73d5eb12-9b39-4f37-9dca-0a3622494f64.png)

I also tried surrounding it by another p tag or div tag but there is an awkward gap between the type and due date
```
<% if (xblockInfo.get('due_date') && !course.get('self_paced')) { %>
...
<% else if (course.get('self_paced') && course.get('is_custom_relative_dates_active') && xblockInfo.get('relative_weeks_due')) { %>
<p>
<span class="icon fa fa-calendar" aria-hidden="true"></span>
...
```

![image](https://user-images.githubusercontent.com/60379333/127563110-7c41e30b-a655-4d78-b743-33f23d49186d.png)

Reply:
Alright. I appreciate you looking into all of these cases! Good to go from my perspective